_data/_events.json

```diff
@@ -0,0 +1,239 @@
+[
+   {
+      "name":"",
+      "id":"",
+      "title":"",
+      "description":"",
+      "callsForPapersDeadline":"",
+      "location":"",
+      "startDate":"",
+      "endDate":"",
+      "date":"",
```

Comment:
Do we need this in addition to startDate?

Reply:
I added a separate ```date``` for cases where we specify the date in words. 
![Screenshot from 2024-01-05 11-16-51](https://github.com/machinetranslate/machinetranslate.org/assets/94985882/5e93a986-8d83-47fc-a624-0f9cfc73f039)

Reply:
Hi!

I'll take a look.

Reply:
Hi @cefoo, 

Since we are auto-generating the events, there wasn't a straightforward way to place the WMT24 event manually at the top. However, I provided an approximate date for it based on the rest of the WMT events so it can be at the top until the actual date becomes known. 

Please let me know if that solution is ok.

Reply:
Edit: I actually added it at the very top.

Reply:
@liashahnazaryan has just created the WMT24 page, with date on December.

However, it would be good to enforce WMT to be on top in future years, so this could be a good practice moving forward. WMT is usually on December, I guess we could just put December 31st until we know the real date. However, that date should be hidden until we know the real one.